### DISCUSSION TITLE
UX for Comment feature on GitBook spaces is terrible

### DISCUSSION THREAD

**Discussion by dot-alert** [GitbookIO/community#208] [0. General bugs and questions]
### Ask a question, start a conversation, or make an annoucement

Hello Team,

Just sharing some feedback on the Comment feature that is built in GitBook.

For a starter, we have been using GitBook since August 2022, which means that we have seen a lot of "upgrades" being deployed on the platform over the past 9+ months.

One of the biggest letdown for us, as a fully remote team that uses GitBook as a collaborative tool, is the very poor experience provided by the new version of the Comment feature. 

Problems are manifold:
- We can't write paragraphs because pressing the "Enter key" generates a new comment.
- We can't view comments independently, because they are all displayed as a list. 
- We can't access the latest comments straightaway, because we have to scroll through all open comments.
- We don't get notifications about replies unless we get specifically tagged into a comment.
- We have to fiddle around with side panels, stacked text boxes just to post new comments.

AFAWK, none of these issues existed in the previous version of the Comment feature, which makes us wonder: "What are the actual benefits of the new version for users?"

TL;DR: It would help if the GitBook team could revert to the original version of the Comment feature and get rid of the current one.

Many thanks.

**Comment by Anshuman71** [comment]
Hi @dot-alert, I built [docsly.dev](https://www.docsly.dev) to provide better feedback collection and collaboration on your documentation.

You get the highlighted text on the comment, see where the user left the comment, reply in-line to the comment, get notifications and summary about these comments/replies as well.
